person.rb

```diff
@@ -1,11 +1,14 @@
-class Person
+require './nameable'
```

Comment:
Please make sure you import the `decorator.rb` file as well so all of the commands can be executed in a single file. I did so locally and your program works as intended, so your project will be ready for approval once you do this.